powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationHistoryEntity.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2017 Lime - HighTech Solutions s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.getlime.security.powerauth.app.server.database.model.entity;
+
+import io.getlime.security.powerauth.app.server.database.model.ActivationStatus;
+import io.getlime.security.powerauth.app.server.database.model.ActivationStatusConverter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+/**
+ * Entity representing activation history used for storing activation status changes.
+ *
+ * @author Roman Strobl, roman.strobl@lime-company.eu
+ */
+@Entity(name = "pa_activation_history")
+public class ActivationHistoryEntity implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
We should probably add a `@SequenceGenerator` annotation, so that we do not use the default `hibernate_sequence` on Oracle. We should also fix the SQL script for Oracle to use another sequence. Issue is related to #7.

Reply:
(we do not need to fix #7, we just should not make the problem more pressing...)